velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -73,6 +74,12 @@ class HdfsFileSystemTest : public testing::Test {
   }
 };
 
+enum WriteErrorType {
+  WRITE_FILE_NOT_EXIST = 1,
```

Comment:
- Use enum class.
- naming: kWriteFileNotExist

Please, take a moment to review https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md